csrankings-a.csv

```diff
@@ -1206,6 +1207,7 @@ Amr Mohamed,Qatar University,http://www.qu.edu.qa/engineering/computer/faculty/a
 Amr Sabry,Indiana University,http://www.cs.indiana.edu/~sabry,dYGSG4EAAAAJ
 Amrita Chaturvedi,IIT (BHU) Varanasi,http://www.iitbhu.ac.in/cse/index.php/people/faculty.html,zoVJnasAAAAJ
 Amro Awad,North Carolina State University,https://sites.google.com/site/amroawad,jDmd7AoAAAAJ
+Amulya Yadav,Pennsylvania State University,https://ist.psu.edu/directory/auy212,jI3_2koAAAAJ

```

Comment:
Not in EECS. https://www.eecs.psu.edu/departments/cse-faculty-list.aspx

Reply:
Please see my previous comment.